### ISSUE TITLE
warmup is not working correctly

### ISSUE THREAD

**Issue by goldfishl** [goldfishl/acupoint_audio_classifer#1]
```
 warmup_scheduler = torch.optim.lr_scheduler.LambdaLR(optimizer, lr_lambda=[lambda step: (step / exp_config['warmup_steps']) * exp_config['lr'], lambda step: (step / exp_config['warmup_steps']) * exp_config['lr'] * exp_config['head_lr']])
```
warmup_scheduler does not return to exp_config['lr'] at the end of warmup_steps. For example, set 'lr' : 2.5e-4, 'warmup_steps' : 1000. But after warmup_steps ends, lr is 2.5e-6.

And setting the scheduler will reset the initial lr set by the optimizer to 0. This will cause the lr to stay 0 while setting 'warmup_steps' : 0 or 1 to disable warmup.

These are strange behaviors.
